packages/components/src/internal/components/domainproperties/DomainForm.spec.tsx

```diff
@@ -684,4 +685,117 @@ describe('DomainForm', () => {
         expect(form).toMatchSnapshot();
         form.unmount();
     });
+
+    test('using allowImportExport', () => {
+        const domain = DomainDesign.create({});
+
+        const form = mount(
+            <DomainForm
+                domain={domain}
+                onChange={jest.fn()}
+                allowImportExport={true}
+            />
+        );
+
+        expect(form.find('.domain-form-manual-section').length).toEqual(1);
+        expect(form.find('.file-form-formats').text()).toContain('.json');
+
+        form.unmount();
+    });
+
+    test('not using allowImportExport', () => {
+        const domain = DomainDesign.create({});
+
+        const form = mount(
+            <DomainForm
+                domain={domain}
+                onChange={jest.fn()}
+                allowImportExport={false}
+            />
+        );
+
+        expect(form.find('.domain-form-manual-section').length).toEqual(0);
+        expect(form.find('.file-form-formats').length).toEqual(0);
+        expect(form.find('.domain-toolbar-export-btn').length).toEqual(0);
+        expect(form.find('.domain-field-top-noBuffer').length).toEqual(2);
+
+        form.unmount();
+    });
+
+    test('using allowImportExport, field view', () => {
+        const fields = [];
+        fields.push({
+            name: 'key',
+            rangeURI: INT_RANGE_URI,
+            propertyId: 1,
+            propertyURI: 'test',
+        });
+
+        const domain = DomainDesign.create({
+            name: 'allowImportExport field view',
+            description: 'basic list domain form',
+            domainURI: 'test',
+            domainId: 1,
+            fields,
+            indices: [],
+        });
+
+        const form = mount(
+            <DomainForm
+                domain={domain}
+                onChange={jest.fn()}
+                domainFormDisplayOptions={{
+                    hideRequired: true,
+                }}
+                allowImportExport={true}
+            />
+        );
+
+        expect(form.find('.domain-toolbar-export-btn').length).toEqual(1);
+        expect(form.find('.domain-field-top-noBuffer').length).toEqual(2);
+
+        const actionButtons = form.find(ActionButton);
+        expect(actionButtons.length).toBe(3);
+        expect(actionButtons.at(1).prop('disabled')).toBe(false);
```

Comment:
I usually just add a comment saying that the button at index 1 is the export button.

Reply:
Done!

Reply:
Done!